### PR TITLE
docs: add ingress-apisix.md

### PR DESCRIPTION
**Pull request by Yilialinn**
Fixes: #[Add issue number here]

Changes:

Screenshots of the change:

### CODE REVIEW

File: blog/zh/blog/2023/ingress-apisix.md [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+keywords: 
+- Apache APISIX
+- Ingress APISIX
+- etcd
+description: 
+authors:
+  - name: "容鑫"
+    title: "Author"
+    url: ""
+    image_url: ""
```
Comment (yzeng25) [inline blog/zh/blog/2023/ingress-apisix.md]:
```suggestion
    url: "https://github.com/AlinsRan"
    image_url: "https://github.com/AlinsRan.png"
```

**Review by yzeng25 — COMMENTED** [review]
The file path should be `blog/zh/blog/2023/10/17/ingress-apisix.md`

File: blog/zh/blog/2023/ingress-apisix.md [hunk 2]
```diff
@@ -0,0 +1,206 @@
+---
+title: "Ingress APISIX：一种无需 etcd 的轻量级部署方式"
+keywords: 
+- Apache APISIX
+- Ingress APISIX
+- etcd
+description:
```
Comment (yzeng25) [inline blog/zh/blog/2023/ingress-apisix.md]: I suggest insert `keywords` and `description` parts between `authors` and `tags`.

```
title
authors
keywords
description
tags
image
```

File: blog/zh/blog/2023/ingress-apisix.md [hunk 3]
```diff
@@ -0,0 +1,206 @@
...
+
+4. **实现 Gatewa API 遇到阻碍**：Gateway API 能够动态的管理一组 Gateway（APISIX） 的生命周期，APISIX 配置主要来源于 etcd，Ingress Controlle 需要同时感知一组 etcd 集群与 APISIX 的状态，这使得维护和管理都会十分复杂。
+
+可以看到，在整个架构中，Apache APISIX 并不依赖于 Ingress Controller，后者只承担着配置推送的角色，实际上它并不具备管理 APISIX 的能力，这些问题在现有的架构中难以解决。**为了这些解决问题，我们需要设计新的 Ingress APISIX 架构。**
+
+## 新架构的设计
+
+如果要解决现有架构的问题，则必须把 etcd 组件去掉。当前主要考虑了以下两个方案：
+
+1. **渲染 apisix.yaml 配置文件**：根据 CRD 生成 `apisix.yaml` 配置文件，APISIX 在 yaml 部署模式下定期每秒钟全量读取 `apisix.yaml` 配置文件。
```
Comment (yzeng25) [inline blog/zh/blog/2023/ingress-apisix.md]:
```suggestion
1. **渲染 `apisix.yaml` 配置文件**：根据 CRD 生成 `apisix.yaml` 配置文件，APISIX 在 yaml 部署模式下定期每秒钟全量读取 `apisix.yaml` 配置文件。
```

File: blog/zh/blog/2023/ingress-apisix.md [hunk 4]
```diff
@@ -0,0 +1,206 @@
...
+
+在之前的版本中，部署 Ingress APISIX 集群需要额外维护一套高可用的 etcd 集群。实际上在使用过程中，它并不是那么好用，会带来了一些问题：
+
+1. **etcd 集群维护成本高**：部署高可用集群需要较高的学习与维护成本，需要考虑内存等系统资源的消耗。在 Kubernetes 中部署 etcd 集群需要注意的事项较多，往往由于不熟悉 etcd 而无法有效解决问题。还需关心内存等系统资源消耗。
+
+2. **使用成本高**：Ingress APISIX 集群的部署需要三个组件，相比于单个组件的 ingress-nginx 而言，当前架构需要较高的学习和调试成本，十分不易于使用，给初次使用者带来额外的负担。
+
+3. **数据冗余和一致性问题**：即 Kubernetes etcd 保留一份，APISIX ETCD 集群也保留了一份。在使用过程中，往往需要避免两者数据不一致的情况，但由于 APISIX 并不依赖于 Ingress controller，它们之间处于解耦的关系，很难处理和避免这种情况。
+
+4. **实现 Gatewa API 遇到阻碍**：Gateway API 能够动态的管理一组 Gateway（APISIX） 的生命周期，APISIX 配置主要来源于 etcd，Ingress Controlle 需要同时感知一组 etcd 集群与 APISIX 的状态，这使得维护和管理都会十分复杂。
```
Comment (yzeng25) [inline blog/zh/blog/2023/ingress-apisix.md]:
```suggestion
- **etcd 集群维护成本高**：部署高可用集群需要较高的学习与维护成本，需要考虑内存等系统资源的消耗。在 Kubernetes 中部署 etcd 集群需要注意的事项较多，往往由于不熟悉 etcd 而无法有效解决问题。还需关心内存等系统资源消耗。

- **使用成本高**：Ingress APISIX 集群的部署需要三个组件，相比于单个组件的 ingress-nginx 而言，当前架构需要较高的学习和调试成本，十分不易于使用，给初次使用者带来额外的负担。
- **数据冗余和一致性问题**：即 Kubernetes etcd 保留一份，APISIX ETCD 集群也保留了一份。在使用过程中，往往需要避免两者数据不一致的情况，但由于 APISIX 并不依赖于 Ingress controller，它们之间处于解耦的关系，很难处理和避免这种情况。

- **实现 Gatewa API 遇到阻碍**：Gateway API 能够动态的管理一组 Gateway（APISIX） 的生命周期，APISIX 配置主要来源于 etcd，Ingress Controlle 需要同时感知一组 etcd 集群与 APISIX 的状态，这使得维护和管理都会十分复杂。
```

**Review by SkyeYoung — DISMISSED** [review]

**Review by kayx23 — DISMISSED** [review]

**Review by yzeng25 — COMMENTED** [review]
Fix lint
image

**Review by kayx23 — APPROVED** [review]

**Review by AlinsRan — APPROVED** [review]

**Pull request merged**